Makefile

```diff
@@ -6,13 +6,29 @@ SHELL = /usr/bin/env bash -o pipefail
 MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 
-# VERSION defines the project version for the bundle.
-# Update this value when you upgrade the version of your project.
-# To re-generate a bundle for another specific version without changing the standard setup, you can:
-# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
-# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+# Check for dirty.
+define check_dirty
+	GIT_SHA=$$(git rev-parse HEAD) || { \
```

Comment:
`GIT_SHA` is not being used. This method can be simplified. 

if git does not exists -> dirty = unknown
if `git diff --stat` returns something -> dirty = true
else -> dirty = false

Reply:
Actually, now it is! 

```make
# Check for dirty.
define check_dirty
	GIT_SHA=$$(git rev-parse HEAD) || { \
		GIT_HASH=$${GIT_SHA:-NO_SHA}; \
	}; \ <-- used to check command output
	if [ -z "$$GIT_HASH" ]; then \ <-- if there was command output for git...
		GIT_DIRTY=$$(git diff --stat); \
		if [ -n "$$GIT_DIRTY" ]; then \
			DIRTY="true"; \
		else \
			DIRTY="false"; \
		fi; \
	else \ <-- otherwise if git command did not output i.e. it does not exist
        DIRTY="unknown"; \
	fi; \
	echo $$DIRTY
endef
```